index.js

```diff
@@ -0,0 +1,37 @@
+import Library from './modules/library.js';
+import displaySection from './modules/displaySection.js';
+import { DateTime } from './node_modules/luxon/src/luxon.js';
+
+// let listBooks
+const listBooks = new Library();
+listBooks.className = 'purple';
+
+// ====================== NAVIGATION =========================
+function UpdateTime() {
```

Comment:
- [ ] Please note that as part of this project's [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md) you need to refactor all your traditional functions to the new ES6 arrow syntax, which is not the case for this function, please convert it to an arrow function.